src/modules/packages.c

```diff
@@ -53,6 +53,34 @@ static uint32_t getNumStrings(const char* filename, const char* needle)
     return count;
 }
 
+static void getActiveBranch()
+{
+    FILE* file = fopen("/etc/pacman-mirrors.conf", "r");
+    if(file == NULL)
+        return; // No file or not Manjaro[based]
+
+    char* line = NULL;
+    size_t len = 0, strip;
+    uint_fast8_t match = 0;
```

Comment:
Wouldn't it be better if match is the return type of sscanf?

Reply:
Probably - I was looking for the smallest size I could use.